.gitignore

```diff
@@ -68,3 +68,4 @@ junit.xml
 tsconfig.tsbuildinfo
 **/public
 **/public
+.idea
```

Comment:
@zuffik I am curious about what this change is for?

Reply:
To prevent config files from jetbrains IDEs (mine is WebStorm) to being tracked in git.